Makefile

```diff
@@ -38,7 +38,8 @@ GOROOT:=
 GO111MODULE=on
 export PATH GOROOT GO111MODULE
 # work around broken PATH export
-SHELL:=env PATH=$(PATH) $(SHELL)
+SPACE:= $(empty) $(empty)
```

Comment:
What do you think about `$(subst ,, )`?
It feels a little gross relying on `empty` being unset. https://stackoverflow.com/a/10571900

Reply:
As I hope not too much people will look inside Makefile: I'm fine with subst, but $(empty) is more obvious (despite the variable should make things clear).